courses/templates/course_detail.html

```diff
@@ -1,8 +1,13 @@
 {% extends "course_app_base.html" %}
-{% load static wagtailimages_tags %}
+{% load static wagtailimages_tags wagtailmetadata_tags %}
+
 
 {% block title %}MIT xPro | {{ course.title }}{% endblock %}
 
+{% block seohead %}
+  {% meta_tags model=course.page %}
```

Comment:
is it supposed to be `course.page` or is that a typo? (extra dot in the middle)

Reply:
By default it requires `page` instance. Since we are rendering through django rather than CMS, we have to pass `page` instance to load metadata.